src/libs/numberFormat/index.native.js

```diff
@@ -0,0 +1,14 @@
+// we only need polyfills for Mobile.
```

Comment:
I thought we only needed it for Android, since iOS already supported it out of the box?

Reply:
Hmm. As per this, a polyfill is implemented 
> Everything in the ES2020 Internationalization API spec (https://tc39.es/ecma402).

I was not sure if IOS has the same behavior. Sorry I was not able to test it on IOS. We can disable it if IOS has them.

Reply:
Can you test it in iOS without the polyfill and see if it works please? There should be instructions on how to test on iOS simulator in the readme

Reply:
That's a bit of a problem. I don't have a Mac but let me see what can I do.

Reply:
Ok, I tested it on IOS. You are right. IOS does have Intl support. Check the snap [here](https://snack.expo.io/@rajat_parashar/number-format)